components/usage-api/usage/v1/usage.proto

```diff
@@ -49,5 +49,6 @@ message BilledSession {
     google.protobuf.Timestamp start_time = 9;
     google.protobuf.Timestamp end_time = 10;
 
-    int64 credits = 11;
+    int64 credits_old = 11 [deprecated=true];   // insufficient precision to represent credits for workspace time
```

Comment:
A rename is a valid backwards compatible operation (that's due to the field numbering).